mlflow/projects/__init__.py

```diff
@@ -290,6 +302,21 @@ def run(
                 )
                 raise
 
+    if not use_conda:
```

Comment:
The default value of `use_conda` is True. If it's False, that means a user specifies it.

Reply:
A user might call `mlflow.projets.run(..., use_conda=True)`. The warning is not raised in this case.